OWML.Common/IModInputHandler.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using UnityEngine;
+
+namespace OWML.Common
+{
+	public enum RegistrationCode
```

Comment:
public enums should be in separate files
also indents

Reply:
separated

Reply:
and changed to spaces

Reply:
I don't see the change. Did you push?

Reply:
sorry, found a bug while playtesting, took some time to fix